client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -63,13 +64,13 @@ const displaySiteUrl = ( siteUrl: string ) => {
 const VisitDashboardItem = ( { site }: { site: SiteData } ) => {
 	const { __ } = useI18n();
 	return (
-		<PopoverMenuItem href={ getDashboardUrl( site.slug ) }>
+		<PopoverMenuItem href={ getDashboardUrl( site.slug, '/home' ) }>
 			{ __( 'Visit Dashboard' ) }
```

Comment:
Should we use `siteBasePath` here to make this link contextual and consistent with other links in the row? Or would we keep the site name link contextual but make "Visit Dashboard" always pointing to the site's dashboard?